Assets/FishFactory/Scripts/Common/ConveyorController.cs

```diff
@@ -6,10 +6,12 @@ public class ConveyorController : MonoBehaviour
 {
     private enum Direction
     {
-        Forward,
-        Backward,
-        Right,
-        Left,
+        Forward_BlueI,
```

Comment:
What is BlueI, D, etc?

Reply:
Names refering to the color used by unity to indicate movement axis: Forward and Backward on Blue axis, Left and Right on Red axis and Up and Down on Green axis. D and I refers to whether it is an increase or decrease in value along the given axis. If these variable names is not suitable, I can rename back to original, or add a comment in the code explaining the names?

Reply:
Should be good